cms/static/js/views/settings/main.js

```diff
@@ -164,18 +174,41 @@ var DetailsView = ValidatingView.extend({
         $(e.currentTarget).attr('title', currentTimeText);
     },
     updateModel: function(event) {
+        var url, image_name;
         switch (event.currentTarget.id) {
         case 'course-image-url':
             this.setField(event);
-            var url = $(event.currentTarget).val();
-            var image_name = _.last(url.split('/'));
+            url = $(event.currentTarget).val();
+            image_name = _.last(url.split('/'));
             this.model.set('course_image_name', image_name);
             // Wait to set the image src until the user stops typing
             clearTimeout(this.imageTimer);
             this.imageTimer = setTimeout(function() {
                 $('#course-image').attr('src', $(event.currentTarget).val());
             }, 1000);
             break;
+        case 'hero-image-url':
```

Comment:
What did you have in mind for test coverage here? Can you point us to the test that covers the existing `course-image-url` as an example of what you would like to see?